site/docs/v1/tech/identity-providers/samlv2/okta-saml.adoc

```diff
@@ -0,0 +1,107 @@
+
+Click [uielement]#Next#, select one of the radio buttons on the [breadcrumb]#Feedback# tab, and click [uielement]#Finish#. Scroll to the bottom of the following page and click the [uielement]#View SAML setup instructions# button. You will need to provide the information on this page to FusionAuth, so keep it open in a separate tab or window.
+
+image::identity-providers/okta-saml/okta-saml-configuration-details.png[saml configuration details,width=1200,role=bottom-cropped]
+
+Return to the previous window and navigate to the [breadcrumb]#Assignments# tab. Click [uielement]#Assign# and then [uielement]#Assign to People#.
+
+image::identity-providers/okta-saml/okta-saml-assign-to-people.png[assign to people,width=1200,role=bottom-cropped]
+
+Click the [uielement]#Assign# button next to the user you would like to authenticate. Click [uielement]#Save and Go Back#, then [uielement]#Done#.
+
+image::identity-providers/okta-saml/okta-saml-assign-button.png[assign button,width=1200]
+
+== FusionAuth Configuration
+
+From your FusionAuth dashboard, navigate to [breadcrumb]#Settings -> Key Master# and click the downward arrow next to [uielement]#Generate Elliptic#. Then click [uielement]#Import Certificate#.
+
+image::identity-providers/okta-saml/okta-saml-import-certificate.png[import certificate,width=1200]
+
+Give this key a [field]#Name#, for example, `OKTA SAML`. Then, copy the [field]#X.509# certificate from the Okta [breadcrumb]#SAML setup instructions# page that you left open previously. Paste it into the [field]#Certificate# box in the FusionAuth Key Master and [uielement]#Submit#.
+
+image::identity-providers/okta-saml/okta-saml-paste-certificate.png[paste certificate,width=1200]
+
+Navigate to [breadcrumb]#Settings -> Identity Providers# and add a new `SAML v2` identity provider.
+
+image::identity-providers/okta-saml/okta-saml-new-idp.png[new identity provider,width=1200,role=bottom-cropped]
+
+Provide a [field]#Name#, for example, `OKTA SAML`. For the [field]#IdP endpoint#, paste the [field]#Identity Provider Single Sign-On URL# from the Okta [breadcrumb]#SAML setup instructions# page. For the verification key, select from the dropdown the name of the key you just created in the Key Master. Optionally, change the [field]#Button text# to `Login with Okta`.
+
+image::identity-providers/okta-saml/okta-saml-new-idp-settings.png[new idp settings,width=1200]
+
+Make sure your application has the [uielement]#Enabled# switch toggled. Then, navigate to the [breadcrumb]#Options# tab and supply the names that you chose for the [field]#user.Id# and [field]#user.email# attribute statements in Okta.
+
+image::identity-providers/okta-saml/okta-saml-claim-options.png[claim options,width=1200]
+
+Hit [uielement]#Save# and [uielement]#View# the identity provider you just created. Scroll down to the [breadcrumb]#SAML v2 Integration details# section and record these values so you can supply them to Okta.
```

Comment:
```suggestion
Hit [uielement]#Save#. Then [uielement]#view# the identity provider you just created (using the green magnifying glass). Scroll down to the [breadcrumb]#SAML v2 Integration details# section and record these values so you can supply them to Okta.
```

Reply:
Which values do you need to record? All of them? Please list them out here.

Reply:
This has been updated to read: 
"Hit [uielement]#Save# and [uielement]#View# the identity provider you just created. Scroll down to the [breadcrumb]#SAML v2 Integration details# section and record the [field]#Callback URL (ACS)# and [field]#Issuer# values so you can supply them to Okta."